example/spec/parallel/moveGranuleCollections/ChangeGranuleCollectionS3.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const { InvokeCommand } = require('@aws-sdk/client-lambda');
+const { lambda } = require('@cumulus/aws-client/services');
+const fs = require('fs');
+const {
+  deleteS3Object,
+  s3ObjectExists,
+} = require('@cumulus/aws-client/S3');
+const { waitForListObjectsV2ResultCount, addCollections, addProviders } = require('@cumulus/integration-tests');
+
+const { getGranule, deleteGranule, removePublishedGranule } = require('@cumulus/api-client/granules');
+const { constructCollectionId } = require('@cumulus/message/Collections');
+const { deleteExecution } = require('@cumulus/api-client/executions');
+const { setupTestGranuleForIngest } = require('../../helpers/granuleUtils');
+const { waitForApiStatus } = require('../../helpers/apiUtils');
+const { buildAndStartWorkflow } = require('../../helpers/workflowUtils');
+const { loadConfig, createTimestampedTestId, createTestSuffix, createTestDataPath, uploadTestDataToBucket } = require('../../helpers/testUtils');
+
+describe('when ChangeGranuleCollectionS3 is called', () => {
+  let stackName;
+  let config;
+  let inputPayload;
+  let provider;
+  let testDataFolder;
+  let granuleId;
+  let startingFiles;
+  let finalFiles;
+  let collection;
+  let targetCollection;
+  let ingestExecutionArn;
+  let cleanupCollectionId;
+  afterAll(async () => {
+    try {
+      await removePublishedGranule({
+        prefix: config.stackName,
+        granuleId,
+        collectionId: cleanupCollectionId,
+      });
+      let cleanup = finalFiles.map((fileObj) => deleteS3Object(
+        fileObj.bucket,
+        fileObj.key
+      ));
+      cleanup.concat(startingFiles.map((fileObj) => deleteS3Object(
+        fileObj.bucket,
+        fileObj.key
+      )));
+      cleanup = cleanup.concat([
+        deleteExecution({ prefix: config.stackName, executionArn: ingestExecutionArn }),
+        deleteGranule({ prefix: config.stackName, granuleId: granuleId }),
+      ]);
+
+      await Promise.all(cleanup);
+    } catch (error) {
+      console.log('cleanup failed with error', error);
+    }
+  });
+  beforeAll(async () => {
+    const inputPayloadFilename = './spec/parallel/ingestGranule/IngestGranule.input.payload.json';
```

Comment:
We shouldn't use a payload from another test, replication is fine here, or moving it to test-data.